_posts/2022-06-10-creat_cf_cap_project_on_sap_bas.md

```diff
@@ -0,0 +1,126 @@
+---
+layout: post
+title: How to create cf cap project on SAP Business Application Studio?
+categories: [SAP CAP & SAP Business Application Studio]
+author_github: nevermore17
+author: Verbin Kirill
+---
+
+In this blog, I’ll explain how to create a cap project on SAP Business Application Studio.
+
+## Create Development space
+
+Open SAP Business Application Studio and create new Dev Space.
+
+![Step 1](../article_images/create_cf_cap_project_on_sap_bas/1.png)
+
+Enter a name for the workspace and select the type `Fill Stack Cloud Application`.
+
+![Step 2](../article_images/create_cf_cap_project_on_sap_bas/2.png)
+
+Now we create our dev space and we can start creating our application.
+
+![Step 3](../article_images/create_cf_cap_project_on_sap_bas/3.png)
+
+## Create simple CAP Project
+
+Now we create a project from template.
+
+![Step 4](../article_images/create_cf_cap_project_on_sap_bas/4.png)
+
+Select the CAP project.
+
+![Step 5](../article_images/create_cf_cap_project_on_sap_bas/5.png)
+
+We will be deploying the application on SAP BTP and HANA Cloud so we need to select the appropriate settings.
+
+![Step 6](../article_images/create_cf_cap_project_on_sap_bas/6.png)
+
+We need to create files service.cds and service.js in folder srv/ and file schema.cds with simple code in folder db for CAP project.
```

Comment:
We need to create files `service.cds` and `service.js` in `srv/` folder and file `schema.cds `with simple code in `db/` folder for CAP project.